spec/suma/async/jobs_spec.rb

```diff
@@ -358,4 +358,34 @@
       end.to perform_async_job(Suma::Async::TopicShim)
     end
   end
+
+  describe "MemberOnboardingVerifiedDispatch" do
+    it "dispatches an SMS to the member preferred messaging" do
+      member = Suma::Fixtures.member(phone: "12223334444").create
+
+      expect do
+        member.onboarding_verified_at = Time.now
+        member.save_changes
+      end.to perform_async_job(Suma::Async::MemberOnboardingVerifiedDispatch)
+
+      expect(Suma::Message::Delivery.all).to contain_exactly(
+        have_attributes(
+          template: "onboarding_verification",
+          transport_type: "sms",
+          to: "12223334444",
+        ),
+      )
+    end
+
+    it "noops if member is already verified" do
```

Comment:
This is testing the `from(nil)` but you really want to test the `return unless member.onboarding_verified?`. This spec isn't really testing the `from(nil)` right either (and it's not needed).